.travis.yml

```diff
@@ -2,9 +2,9 @@ language: ruby
 
 rvm:
   - 1.9.3
-  - 2.0.0
-  - 2.1.5
-  - 2.2.0
+  - 2.0
```

Comment:
2.0.x still has the patch levels, so I think we can only do this for 2.1 and 2.2